crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -41,6 +41,19 @@ def roles_for_select(selected)
     )
   end
 
+  # Possible values for nodes default file system
+  def fs_for_select(selected)
+    options_for_select(
+      [
+        ["ext3", "ext3"],
```

Comment:
@vuntz do you think we should say about some filesystem that is/is not supported?